ep/build-velox/src/get_velox.sh

```diff
@@ -17,7 +17,7 @@
 set -exu
 
 VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2024_11_12
+VELOX_BRANCH=2024_11_12_1
```

Comment:
Is this intended change?

Reply:
Yes, it avoids pushing to the previous branch.